Add subctl benchmark cmd documentation (#274)
submariner-io/submariner-website: src/content/deployment/subctl/_index.en.md

| `--only`                            | Comma separated list of specific verifications to perform
| `--enable-disruptive`               | Enable verifications which are potentially disruptive to your deployment

### benchmark

#### benchmark throughput

`subctl benchmark throughput <kubeconfig1> <kubeconfig2>`

> The `benchmark throughput` command runs a throughput benchmark test between two clusters.

Reviewer: ```suggestion
The `benchmark throughput` command runs a throughput benchmark test between two specified clusters.
```

When running `benchmark throughput` command, two type of tests will be issued:

> * Pod to Pod - gateway to gateway

Reviewer: ```suggestion
* Pod to Pod - where both Pods are scheduled on Gateway nodes
```

> * Pod tp Pod - non-gateway to non-gateway

Reviewer: ```suggestion
* Pod to Pod - where both Pods are scheduled on non-Gateway nodes
```

> * Pod tp Pod - where both Pods are scheduled on non-Gateway nodes

Reviewer: ```suggestion
* Pod to Pod - where both Pods are scheduled on non-Gateway nodes
```

#### benchmark latency

`subctl benchmark latency <kubeconfig1> <kubeconfig2>`

The `benchmark latency` command runs a latency benchmark test between two clusters.
When running `benchmark latency` command, two type of tests will be issued:

> * Pod to Pod - gateway to gateway

Reviewer: ```suggestion
* Pod to Pod - where both Pods are scheduled on Gateway nodes
```

> * Pod tp Pod - non-gateway to non-gateway

Reviewer: ```suggestion
* Pod to Pod - where both Pods are scheduled on non-Gateway nodes
```

> * Pod tp Pod - where both Pods are scheduled on non-Gateway nodes

Reviewer: ```suggestion
* Pod to Pod - where both Pods are scheduled on non-Gateway nodes
```
